### DISCUSSION TITLE
Disc Size

### DISCUSSION THREAD

**Discussion by OneGameDad** [WaRtr0/francinette-image#4] [Ideas]
Francinette can be quite big and given we're only allocated 5GB of space (at least at Hive Helsinki), it would be helpful to note the ultimate install size in the install instructions.

**Comment by WaRtr0** [comment]
Yes, it's a very good idea, I'm going to set this up as soon as possible, just until I recover the size of the docker image. I recommend that you always try to use the original “francinette” which is extremely lighter, but where you can have compatibility problems depending on the campuses, for example with the permissions you have on your sessions, or the binary versions. The “francinette-image” is very heavy because it installs a second linux system “ubuntu:22” with all the packages required to install the original “francinette”.

Thank you for your contribution
